app/scripts/controllers/swaps.js

```diff
@@ -101,19 +112,48 @@ export default class SwapsController {
     this._setupSwapsLivenessFetching()
   }
 
+  // Sets the refresh rate for quote updates from the MetaSwap API
+  async _setSwapsQuoteRefreshTime() {
+    // Default to fallback time unless API returns valid response
+    let swapsQuoteRefreshTime = FALLBACK_QUOTE_REFRESH_TIME
+    try {
+      swapsQuoteRefreshTime = await this._fetchSwapsQuoteRefreshTime(
+        FALLBACK_QUOTE_REFRESH_TIME,
+      )
+    } catch (e) {
+      this._trackEvent({
```

Comment:
I feel a little guilty saying this since this was my suggestion, but we may want to revisit this in a future PR instead of adding it now. We should sync with Lorenzo first about what this event schema should look like, and ensure it's tracked in Segment. Maybe we can create a general-purpose error event schema so that it's easier to add more in the future as well.